assets/js/base/components/formatted-monetary-amount/index.tsx

```diff
@@ -27,27 +32,32 @@ const currencyToNumberFormat = ( currency ) => {
 	};
 };
 
+interface FormattedMonetaryAmountProps {
+	className?: string;
+	value: number;
+	currency: Currency;
+	onValueChange?: ( unit: number ) => void;
+}
+
 /**
  * Formatted price component.
  *
  * Takes a price and returns a formatted price using the NumberFormat component.
  *
  * @param {Object} props Component props.
- * @param {string=} props.className CSS class used.
+ * @param {string} props.className CSS class used.
```

Comment:
We can remove this docblock and inline the comments now (within `FormattedMonetaryAmountProps`)